spec/features/cart/show_spec.rb

```diff
@@ -55,6 +55,61 @@
 
         expect(page).to have_content("Total: $124")
       end
+
+	it 'I can increase quantity in cart' do
+		visit '/cart'
+		within "#cart-item-#{@paper.id}" do
+			click_on "+1"
+			expect(page).to have_content("2")
+			click_on "+1"
+			expect(page).to have_content("3")
+		end
+	end		
+
+	it 'I can decrease quantity in cart' do
```

Comment:
I think we could include a sad path test so that they cannot increase the quantity in the cart above what the merchant has in inventory?

Reply:
yeah, it was just getting late